### DISCUSSION TITLE
Updating within v5.x

### DISCUSSION THREAD

**Discussion by dannycam212121** [pimcore/pimcore#16440] [Q&A / Support]
In the step-by-step process to update Pimcore to version 5.4 and above held in this URL [https://pimcore.com/docs/pimcore/5.8/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/Within_V5/Update_from_5.x_to_5.4_or_above.html](url) , the GitHub URLs for the last steps don't work:
`wget https://raw.githubusercontent.com/pimcore/skeleton/master/bin/console -O bin/console
wget https://raw.githubusercontent.com/pimcore/skeleton/master/web/app.php -O web/app.php
wget https://raw.githubusercontent.com/pimcore/demo-basic/master/app/AppKernel.php -O app/AppKernel.php`
I've tried to retrieve it from any single place that I could, but it wasn't possible at all. Does anyone know where I could get them?

**Comment by kingjia90** [comment]
Replace `master` with `v1.3.0` tag (by looking at the composer, it was the last version that required pimcore v5) eg.https://github.com/pimcore/skeleton/blob/v1.3.0/web/app.php

**Reply by dannycam212121** [reply]
Before updating those files, I've encountered another error while executing `php manual-migration.php 235`:
`Declaration of Symfony\Bridge\Monolog\Handler\ConsoleHandler::handle(array $record) must be compatible with Monolog\Handler\AbstractProcessingHandler::handle(array $record): bool in`
I've done everything step-by-step but this error keeps appearing when I try to execute migration for build 280.
Do I have to update symfony (using v 3.4.35 now) or something like that?

If that helps, attached is my composer.json file I'm trying to use:
[composer.json](https://github.com/pimcore/pimcore/files/13936703/composer.json)

And this the `migration.php` script step that causes the error (fully available on https://gist.githubusercontent.com/brusch/c3e572947a7a7e8523e18e9787cf88c3/raw/pimcore-migrations-40-to-54.php):

```
$migrations[280] = function () {
    //setting replyTo = null if default empty value exists
    $emailLogs = new \Pimcore\Model\Tool\Email\Log\Listing();
    $emailLogs->setCondition("replyTo = ''");
    foreach ($emailLogs->load() as $ekey => $emailLog) {
        $emailLog->setReplyTo(null);
        $emailLog->save();
    }

};
```

**Reply by kingjia90** [reply]
unfortunately i am unable to provide any help in here, unfortunately the v5 is quite old and outdated and i actually don't have any experience in there (i didn't even know Pimcore existence at the time 😄 )
